.github/RELEASE_CHECKLIST.md

```diff
@@ -47,6 +47,20 @@
 6. Publish release on GitHub.
 7. Announce it on Slack.
 
+## Release FerretDB/documentdb
```

Comment:
That part should be in the documentdb repo. But we also should update the list above: we want to upload DcoumentDB `.deb` packages for that version of FerretDB to the FerretDB release page

Reply:
Added it in documentdb repo https://github.com/FerretDB/documentdb/pull/9 :+1: